Update verified.mdx (#338)
hashicorp/terraform-docs-common: website/docs/registry/modules/verified.mdx

---
page_title: Terraform Registry - Partner Modules
description: >-
  Partner modules are actively maintained by partner
  contributors.
---

# Partner Modules

> Partner modules are published by tech and cloud organizations that have a partnership agreement with HashiCorp. Partner contributors actively maintain their modules to stay up-to-date and compatible with both Terraform and their respective providers.

Reviewer: Suggesting a rewrite of the first sentence to be in active voice: 
```suggestion
Organizations that have a partnership agreement with HashiCorp can publish partner modules. Partner contributors actively maintain their modules to stay up-to-date and compatible with both Terraform and their respective providers.
```

Reviewer: Do we want to mention or link to the [Partner program](https://www.hashicorp.com/partners/become-a-partner?ajs_aid=b4683bed-199f-451d-ad25-9546dbd0ec8f&product_intent=terraform) here? No worries if not, I'm just curious.

The partner badge appears next to modules that are published by a partner.

> ![Verified module listing](/img/docs/registry-verified.png)

Author: also needs a new image

> Partner modules are expected to be actively maintained by HashiCorp partners.

Reviewer: Rewrite in active voice and suggested we remove the "expected to be" because it feels vague:
```suggestion
HashiCorp partners actively maintain partner modules.
```

> The partner badge isn’t indicative of flexibility or feature support; very

Reviewer: Breaking this down into lines (we try to avoid semicolons in documentation):
```suggestion
The partner badge doesn't indicate flexibility or feature support. Simple
```

Reviewer: I feel like the middle two sentences "Simple modules can have the badge just because they're great examples of modules. Likewise, an non-partner module could be extremely high quality and actively maintained." doesn't really need to be said here. This extra elaboration feels a bit unnecessary after the explicit "The partner badge doesn't indicate flexibility or feature support" statement, how do you feel about it?

simple modules can have the badge just because they're great examples of modules.
Likewise, an non-partner module could be extremely high quality and actively

> maintained. An non-partner module shouldn't be assumed to be poor quality, it

Reviewer: ```suggestion
maintained. Refrain from assuming that a non-partner module is worse than a partner one.
```

> only means it hasn't been created by a HashiCorp partner.

Reviewer: Edited for active voice:
```suggestion
The non-partner designation merely means a HashiCorp partner did not create that module.
```

> When [using registry modules](/terraform/registry/modules/use), there is no

Reviewer: ```suggestion
You can use both [partner and non-partner registry modules](/terraform/registry/modules/use) in the same way.
```

> difference between a partner and non-partner module; they are used the same

Reviewer: Suggest removing these final lines in favor of the sentence above:
```suggestion
```

> way.

Reviewer: Suggest removing these final lines in favor of the sentence above:
```suggestion
```
